### DISCUSSION TITLE
Add @kenafoster to Contributors team

### DISCUSSION THREAD

**Discussion by tylergraff** [nebari-dev/nebari#2491] [Community]
I nominate @kenafoster to the Contributors team
Ken currently has 5 contributions merged to the Nebari repo:  
https://github.com/nebari-dev/nebari/pulls?q=is%3Apr+author%3Akenafoster+is%3Amerged+

**Comment by dcmcand** [comment]
@kenafoster has been involved and has a history of contributions to the project. I approve of adding him to the team. If anyone from @nebari-dev/core-team has comments, please add them. Otherwise we will add him.

**Comment by dcmcand** [comment]
@dharhas can you add @kenafoster to the contributers team?

**Comment by dharhas** [comment]
invitation sent.
